.internal/baseSum.js

```diff
@@ -1,3 +1,5 @@
+import baseToNumber from '.internal/baseToNumber.js'
```

Comment:
Let's rather prefix `_internal` as it might cause issues with `.` in linux or other OS systems